scripts/installer/utils/extractors.py

```diff
@@ -1,161 +1,52 @@
-import ast
+from datetime import datetime
 import json
-
-from installer.utils.logger import logger
+import os
+import subprocess
+import tempfile
 
 ## ------------------------------------------------------------------------------------
 ## Convert terraform.tfvars to JSON
 ## Notes:
-##   1. I know it's dumb to home-roll your own parser, but I don't want to introduce extra random
-##      packages from the internet. This is fine for our purposes.
-##   2. Clean command: `sed '/^\s*\/\*/,/^\s*\*\//d;/^\s*#/d' terraform.tfvars > terraform_no_comments.tfvars`
-##      However, `sed` has problem on MacOS, so sticking with (uglier) native Python.
+##   1. Deprecation Notice: Starting May 16, 20205, the bespoke parser to convert terraform.tfvars into json will be replaced with a new container based solution from tmccombs/hcl2json. The home-rolled parser was originally created to avoid introducing packages from the internet. Due to more complicated parsing needs, the new parser solution has been implemented. To refer to the previous parser, please refer to previous Git histroy.
```

Comment:
Since we just introduced the deprecation notice, I don't think we should be deleting the old function yet (_since this would make this not a deprecation notice but rather an after-the-fact deletion notice).

With that said, nobody uses this function but us and I can't see any reason for us to go back to my crappy parser rather than the much better new solution. 

I'm inclined to keep the old code deleted and just rewrite the comment a bit to explain why we nuked the old code and what the implications of the new solution are (_I'll write up more in the official docs_).

Reply:
Modified in latest commit.